src/Microsoft.VisualStudio.Services.Agent/AgentServer.cs

```diff
@@ -28,6 +31,9 @@ public interface IAgentServer : IAgentService
         // job request
         Task<TaskAgentJobRequest> RenewAgentRequestAsync(int poolId, long requestId, Guid lockToken, CancellationToken cancellationToken = default(CancellationToken));
         Task<TaskAgentJobRequest> FinishAgentRequestAsync(int poolId, long requestId, Guid lockToken, DateTime finishTime, TaskResult result, CancellationToken cancellationToken = default(CancellationToken));
+
+        // ReleaseManagement
+        IEnumerable<AgentArtifactDefinition> GetReleaseArtifactsFromService(Guid teamProject, int releaseId, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
ReleaseManagement should leave in a separate server other than AgentSever, just like we had BuildServer for buildclient